features/command_line/order.md

```diff
@@ -13,6 +13,9 @@ order of groups at each level is randomized.
 
 With `rand` you can also specify a seed.
 
+Use `modification_time` to run the most recently modified files first. You can
+combine it with `--next-failure` to find the most recent failing spec.
```

Comment:
This is a bit confusing to me in several ways.

Firstly, `--next-failure` says it's "Equivalent to `--only-failures --fail-fast --order defined`". However, the new flag is overriding the `defined` order.

Another thing is that the random order is also randomizing examples/groups, not just the files, while the intent of the new flag is to specify the order of the files, and not necessarily dictate to use defined or random order inside the files. I see it as complementing random/defined, not replacing them. Do you think this would be possible?

How does the new flag work with a random seed?

Reply:
I didn't understand how this could work with a random seed. Do you mean running **files** by modification time, but the **specs inside each file** in random order?

Reply:
It actually hit me by surprise that with the random order spec files are also run in random order. And I initially thought that the idea of the new flag was to change the "by load order" as in `defined` strategy to use `mtime` instead.

It looks like that those strategies are "dumb", they don't really care if it's a spec file (top-level example group) or examples from the same example group. So it's not an easy task to teach them to do one thing with specs, and another thing with examples/groups.
I suggest improving the documentation to make it clear how it all works, including the combination of the new flag with `--order`. I guess there's no warning that `--order` and the new flag (sorry for calling it like I'm a Silence of the lambs maniac, it's because we haven't settled on its name yet) are mutually exclusive right now, and they are mutually exclusive, right?.

Side observation: it's also surprising to me that if a file has several top-level example groups, `random` order can run them interleaved with examples from another file.

Reply:
I've updated the description. WDYT?